src/TwigComponent/README.md

```diff
@@ -13,17 +13,13 @@ Every component consists of (1) a class:
 // src/Components/AlertComponent.php
 namespace App\Components;
 
-use Symfony\UX\TwigComponent\ComponentInterface;
+use Symfony\UX\TwigComponent\Attribute\TwigComponent;
 
-class AlertComponent implements ComponentInterface
+#[TwigComponent('alert')]
```

Comment:
this should follow the established convention and be named `AsTwigComponent`

Reply:
Actually, I don't really understand why it's coupled with twig. The nice thing about attributes vs interfaces it that it removes coupling and allows a generic way to configure metadata. What about `AsComponent`?

Reply:
But in practice, the library that provides this component IS coupled to Twig (purposely). I'm also worried that `AsComponent` is a little generic (that component word is loaded).